### ISSUE TITLE
[csharp] compile.mustach does not respect packageName

### ISSUE THREAD

**Issue by markebbers** [swagger-api/swagger-codegen#5239]
##### Description

The generated build.bat file, by the compile.mustach template, always uses the packageName IO.Swagger.

##### Swagger-codegen version

Tested with 2.2.0 and latest Master branch

##### Command line used for generation

Just set the option packageName.

##### Steps to reproduce

Just generate Csharp code with a custom packageName.

##### Suggest a Fix

Tried to push a fix but not allowed :) Change latest line from csharp/compile.template to
```
/target:library /out:bin\{{packageName}}.dll /recurse:src\{{packageName}}\*.cs /doc:bin\{{packageName}}.xml
```

**Comment by ePaul** [comment]
Thanks for the issue (and your suggested fix).

The way to propose changes to Open-Source projects is via pull requests.
Create a fork of the repository, push your changes there (preferentially to a new branch named in a way we can see what is it about), then open a pull-request here.

For details, see [CONTRIBUTING.md](https://github.com/swagger-api/swagger-codegen/blob/master/CONTRIBUTING.md).

**Comment by wing328** [comment]
@markebbers thanks for the suggested fix. You will need to first "fork" the repo by clicking on the "Fork" button. Then in your local machine, run the following

1) git clone https://github.com/markebbers/swagger-codegen.git
2) cd swagger-codegen
3) git checkout -b fix_issue9999
4) make changes
5) git commit -a (you may need to use `git add filename` to add new files)
6) git push origin fix_issue9999
7) Visit https://github.com/swagger-api/swagger-codegen in your browser and click on the button to file a new PR based on fix_issue9999

**Comment by wing328** [comment]
@markebbers The issue should be fixed by @jaceee via #5264. Please pull the latest master to give it a try.
